Source/Mosa.Utility.BootImage/Limine.cs

```diff
@@ -18,7 +18,8 @@ enum CacheState { Clean, Dirty }
 
 		private const int BlockSize = 512;
 
-		public static void Deploy(string path)
+		// TODO: Fix for VDI
```

Comment:
is TODO still applicable?

Reply:
It is, because Limine (if used with VDI image) will overwrite its header (since the VDI header starts at 0, and Limine also starts at 0).